test/run-drun/ok/optimise-for-array-eff.ic-ref-run.ok

```diff
@@ -0,0 +1,18 @@
+→ update create_canister(record {settings = null})
+← replied: (record {hymijyo = principal "cvccv-qqaaq-aaaaa-aaaaa-c"})
```

Comment:
@nomeata just out of curiosity: why `hymijyo`?

Reply:
https://github.com/nomeata/haskell-candid/blob/master/src/Codec/Candid/Hash.hs#L24-L45

Reply:
Fun :-) Thanks!